scripts/gha/test_simulator.py

```diff
@@ -501,19 +509,19 @@ def _setup_android(platform_version, build_tool_version, sdk_id):
     "platforms;%s" % platform_version, 
     "build-tools;%s" % build_tool_version]
   logging.info("Install packages: %s", " ".join(args))
-  subprocess.run(args=args, check=True)
+  _run_with_retry(args)
 
   command = "yes | sdkmanager --licenses"
   logging.info("Accept all licenses: %s", command)
-  subprocess.run(command, shell=True, check=False)
+  _run_with_retry(command, shell=True, check=False)
 
   args = ["sdkmanager", sdk_id]
   logging.info("Download an emulator: %s", " ".join(args))
-  subprocess.run(args=args, check=True)
+  _run_with_retry(args)
 
   args = ["sdkmanager", "--update"]
   logging.info("Update all installed packages: %s", " ".join(args))
-  subprocess.run(args=args, check=True)
+  _run_with_retry(args, check=False)
```

Comment:
Why changing check from True to False?

Reply:
This is a non-critical step.
`check=True` only when this step block next steps.